modal.js

```diff
@@ -0,0 +1,196 @@
+
+    // liveUrl:
+    // sourceUrl:
+  },
+
+  {
+    projectTitle: 'Multi-Post Stories',
+
+    projectImage: 'images/car-cruise.png',
+
+    projectSubtitle: 'Gain+Glory',
+
+    skills: ['Ruby on rails', 'Jsx', 'React'],
+
+    href: 'https://github.com/Eternalgratis/car_cruise_rental',
+
+    // liveUrl:
+    // sourceUrl:
+  },
+
+  {
+    projectTitle: 'Multi-Post Stories',
+
+    projectImage: 'images/awesomebooks.png',
+
+    projectSubtitle: 'Gain+Glory',
+
+    skills: ['JavaScript', 'CSS', 'HTML'],
+
+    href: 'https://github.com/Eternalgratis/AwesomebooksES6',
+
+    // liveUrl:
+    // sourceUrl:
+  },
+];
+
+// Adding projects section
+
+function showProj() {
+  let projCard = '';
+  for (let projIndex = 0; projIndex < proj.length; projIndex += 1) {
+    const worksection = `
+  
+    <div class='Projects'>
+                <div class='empty-div'>
+                <img src='${proj[projIndex].projectImage}' alt='project image' class='portfolio'>
+                </div>
+                <div class='Proj-text1'>
+                    <h3>${proj[projIndex].projectTitle}</h3>
+                    <h3>${proj[projIndex].projectSubtitle}</h3>
+                    <ul class='pro-tech'>
+                        <li>${proj[projIndex].skills[0]}</li>
+                        <li>${proj[projIndex].skills[1]}</li>
+                        <li>${proj[projIndex].skills[2]}</li>
+                    </ul>
+                    <br>
+                    <a class='pro-btn' href='${proj[projIndex].href}'>See Project</a>
+                </div>
+            </div>
+  `;
+
+    projCard = projCard.concat(worksection);
+  }
+
+  return projCard;
+}
+
+const projHTML = showProj();
+
+const worksSection = document.getElementById('works-section');
+worksSection.insertAdjacentHTML('beforeend', projHTML);
+const modal = document.querySelector('.Modal');
+
+// Modal popup
+const showModal = () => {
+  modal.classList.add('active');
+};
+
+const seeProjectBtns = document.querySelectorAll('.pro-btn');
+
+seeProjectBtns.forEach((probtn) => {
+  probtn.addEventListener('click', showModal);
+});
+
+const closeContainer = document.querySelector('.close-container');
+closeContainer.addEventListener('click', () => {
+  modal.classList.remove('active');
+});
+
+// Form Validation
+const form = document.querySelector('.contact-form');
+const formEmail = form.elements.email;
+const formErrorMessage = document.querySelector('.error-message');
+
+form.addEventListener('submit', (event) => {
+  if (formEmail.value.match(/[A-Z]/)) {
+    event.preventDefault();
+    formErrorMessage.classList.remove('hide');
+  }
+});
+
+function sendFormData() {
+  const formData = new FormData();
+
+  const xhr = new XMLHttpRequest();
+  xhr.open('POST', 'https://formspree.io/f/mpzbbknz', true);
+  xhr.onload = function () {
+    if (xhr.status === 200) {
+      alert('Form data sent!');
+    } else {
+      alert('Error sending form data');
+    }
+  };
+
+  xhr.send(formData);
```

Comment:
- [ ] Your sendformData method currently does not send anything, I suggest you give the HTML ``form`` as an argument in the FormData, as this will help get form inputs into the formData object so that you can send it to Formspree.  Here is a article to help [FormData](https://developer.mozilla.org/en-US/docs/Web/API/FormData/FormData) 😉